cadc-file-validate/cadc-file-validate.properties

```diff
@@ -0,0 +1,29 @@
+# Sample configuration file.
```

Comment:
Please don't commit sample config files. Devs can cut & paste from docs and **must edit** their own. Ops will be generating them via some TBD mechanism from something TBD.

Yeah, that's super-vague :-)